samples/nrf9160/coap_client/README.rst

```diff
@@ -3,12 +3,24 @@
 nRF9160: nRF CoAP Client
 ########################
 
-The nRF CoAP Client sample demonstrates how to receive data from a public CoAP server with an nRF9160 SiP.
+The nRF CoAP Client sample demonstrates the communication between a public CoAP server and an nRF9160 SiP which acts as the CoAP client.
```

Comment:
"and an nRF9160 SiP that acts as the CoAP client" or "and an nRF9160 SiP, which acts as the CoAP client" - in this case, both are possible. But not which without comma. ;)

Reply:
Updated.